pipelines/main.go

```diff
@@ -98,7 +103,7 @@ func newService(ctx context.Context, basePath string) (*genomics.Service, error)
 
 	ctx = context.WithValue(ctx, oauth2.HTTPClient, &http.Client{Transport: &transport})
 
-	client, err := google.DefaultClient(ctx, genomics.GenomicsScope)
+	client, err := google.DefaultClient(ctx, genomicsScope)
```

Comment:
You should still use the scope constant from the generated package (eg v2beta.CloudPlatformScope)